nucypher/characters/control/specifications/fields/treasuremap.py

```diff
@@ -39,13 +35,17 @@ def _deserialize(self, value, attr, data, **kwargs):
             raise InvalidInputData(f"Could not parse {self.name}: {e}")
 
     def _validate(self, value):
-
-        splitter = BytestringSplitter(signature_splitter,
-                                  (bytes, HRAC_LENGTH),  # hrac
-                                  (UmbralMessageKit, VariableLengthBytestring)
-                                  )  # TODO: USe the one from TMap
         try:
-            signature, hrac, tmap_message_kit = splitter(value)
-            return True
-        except InvalidNativeDataTypes as e:
-            raise InvalidInputData(f"Could not parse {self.name}: {e}")
+            # Unsigned TreasureMap (Federated)
+            from nucypher.policy.maps import TreasureMap as UnsignedTreasureMap
+            splitter = UnsignedTreasureMap.get_splitter(value)
+            splitter(value)
+        except InvalidNativeDataTypes:
```

Comment:
Why not just always sign the treasure map? Do we still use 65-byte signatures for the blockchain case? If that's a normal signature, everything will be perfectly compatible.

Reply:
> It might be enough to use the stamp instead of the transacting power to sign

We can use the same "trampoline" we had for `IndisputableEvidence`: sign the stamp with transacting power, and stamp the map. But then we'll have to have some mechanism of discovery for these "identity evidences"... Which may be hard, Alice is not Ursula and isn't always online.

Reply:
Opened https://github.com/nucypher/nucypher/issues/2737